cvmfsInstall.sh

```diff
@@ -4,6 +4,7 @@ CMS_WEEK=$2
 RELEASE_NAME=$3
 export BASEDIR=/cvmfs/cms-ib.cern.ch
 export BASEDESTDIR=/cvmfs/cms-ib.cern.ch
+export PROOTDIR=/build/cmsbuild/proot
```

Comment:
I looked and Jenkins job parameters and environment variables, I couldn't find a good reference. I guess, this has to be hardcoded somewhere. We could leave it here and discussing the actual path we want to use, or move it to machine specific environment variable in Jenkins. I probably prefer to keep it in this script.